### PR TITLE
Fix #2809: Add label for AddProfileActivity

### LINKED ISSUE #2809: [A11y] Add label for AddProfileActivity
## Task
Introduce label for `AddProfileActivity` as `Add Profile`.

Reference PR: https://github.com/oppia/oppia-android/pull/2750

When you create a PR for this, make sure you add test to check the functionality. Also, suggest working on this PR only if its your first or second PR to Oppia-Android project.

## General Explanation
When `Talkback` is on, whenever we click on any option which leads to new screen the reader will automatically read the `label` associated with that screen first. If in case label in not present then it will read app name i.e. `Oppia`.

### Example

https://user-images.githubusercontent.com/9396084/108906068-78b75080-7646-11eb-943e-212e29fc3e98.mp4

In this above video as soon as we double-tap `Battery`, new screen is opened. 
The reader first outputs `Battery` followed by `Navigate up, Button` and finally `Battery` again. In this the first `Battery` output is the `label` of this new screen which indicates the learner which screen is opened. The second `Battery` is because we selected the title in the screen and it's just reading that text.
So we need `labels` for various screens general which matches the title but if the title is dynamic then we need static label.

### PR DESCRIPTION
**Pull request by ArpitShukIa**
## Explanation

Fixes #2809: Added label for AddProfileActivity
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by ArpitShukIa** [comment]
@rt4914 PTAL.

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt [hunk 1]
```diff
@@ -130,6 +131,19 @@ class AddProfileActivityTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  @Test
+  fun testAddProfileActivity_hasCorrectActivityLabel() {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt:144]: Suggest using `ActivityTestRule` similar to https://github.com/oppia/oppia-android/blob/9b7d57ecd1f00ed630c1b95d56e4cdfc4c84e8c0/app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt#L144

**Review by rt4914 — CHANGES_REQUESTED** [review]
@ArpitShukIa Suggested change.

File: app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt (hunk 1 above)
Comment (ArpitShukIa) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt:144]: @rt4914 To launch `AddProfileActivity` using `ActivityTestRule`, we need an intent.
`fun createAddProfileActivityIntent(context: Context, colorRgb: Int): Intent` requires a `colorRgb` value.
Other callers of this function call it using:
```
activity.startActivity(
  AddProfileActivity.createAddProfileActivityIntent(
    context, activity.intent.getIntExtra(ADMIN_AUTH_COLOR_RGB_EXTRA_KEY, -10710042)
  )
)
```
Should I pass the default value in the test?

Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt:144]: This reference can help on how to create intent: https://github.com/oppia/oppia-android/blob/9b7d57ecd1f00ed630c1b95d56e4cdfc4c84e8c0/app/src/sharedTest/java/org/oppia/android/app/completedstorylist/CompletedStoryListActivityTest.kt#L455

Comment (ArpitShukIa) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt:144]: @rt4914 Yeah, I saw that. But `createAddProfileActivityIntent()` requires a colorRgb value to create an intent. What should I pass there?

Comment (rt4914) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt:144]: The value is mentioned in the above code that you shared. `-10710042`

Comment (ArpitShukIa) [reply app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt:144]: Done

**Comment by ArpitShukIa** [comment]
@rt4914 PTAL

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Review by BenHenning — APPROVED** [review]
Thanks @ArpitShukIa. LGTM for codeowners.

**Pull request merged**